### PR TITLE
Jmap quotas version 11

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: spec/quotas/quota.mdown [hunk 1]
```diff
@@ -81,27 +83,27 @@ optimise data transfer by keeping track of changes to Quota usage separate from
...
 
-A **FilterCondition** object has the following properties, any of which may be omitted:
+A **FilterCondition** object has the following properties, any of which may be included or  omitted:
 
 * name: "String" 
   
   The Quota _name_ property contains the given string.
 
-* scopes: "Scope[]" 
+* scopes: "Scope"
```
Comment (chibenwa) [inline spec/quotas/quota.mdown:100]: Shouldn't it be 

```suggestion
* scope: "Scope" 
```

File: spec/quotas/quota.mdown [hunk 2]
```diff
@@ -81,27 +83,27 @@ optimise data transfer by keeping track of changes to Quota usage separate from
...
   The Quota _name_ property contains the given string.
 
-* scopes: "Scope[]" 
+* scopes: "Scope" 
   
-  The Quota _scope_ property must be in this list to match the condition.
+  The Quota _scope_ property must match the given value exactly.
 
-* resourceTypes: "ResourceType[]" 
+* resourceTypes: "ResourceType"
```
Comment (chibenwa) [inline spec/quotas/quota.mdown:104]:
```suggestion
* resourceType: "ResourceType" 
```

?

File: spec/quotas/quota.mdown (hunk 1 above)
Comment (Arsnael) [reply spec/quotas/quota.mdown]: Yep I noticed it myself while I was answering mails :) but good catch !

File: spec/quotas/quota.mdown [hunk 3]
```diff
@@ -81,27 +83,27 @@ optimise data transfer by keeping track of changes to Quota usage separate from
...
+Servers MAY decide to add other properties to the list that they judge to be changing frequently.
 
 This method's usage is demonstrated in this document at (#requesting-latest-quota-changes).
 
 ## Quota/query
 
 This is a standard "/query" method as described in [@!RFC8620], Section 5.5.
 
-A **FilterCondition** object has the following properties, any of which may be omitted:
+A **FilterCondition** object has the following properties, any of which may be included or  omitted:
```
Comment (quantranhong1999) [inline spec/quotas/quota.mdown:94]:
```suggestion
A **FilterCondition** object has the following properties, any of which may be included or omitted:
```

**Comment by brong** [comment]
I guess we should merge this since it's been published already!

**Comment by Arsnael** [comment]
> I guess we should merge this since it's been published already!

Last commit wasn't as I was waiting for more replies before redoing a release, but it's ok I will keep track of it ;)

**Pull request merged**